### PR TITLE
Autosys Sensor

### PR DESCRIPTION
**Pull request by rebecca-rajshree**
## Description
Having an Autosys sensor operator in the workflow would help us place an dependency on Autosys jobs when necessary.

## Related Issue
#23 

## Motivation and Context
This would help all the teams having dependencies on teradata tables by checking the Autosys job and making sure the data is up-to-date before ingesting the table.

## How Has This Been Tested?
Ran this on the databricks notebook
I ran 'make check' to align with the formatting standards.
I ran 'make test' and everything was successful except the test_cron_conversion function.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by rebecca-rajshree** [comment]
Testing in progress

**Comment by rebecca-rajshree** [comment]
Integration Testing (Tested the sensor as a part of a workflow to make sure the sensor works as expected) is succeeful

**Comment by rebecca-rajshree** [comment]
Screenshot 2023-09-22 at 13 40 50

File: pyproject.toml [hunk 1]
```diff
@@ -79,7 +79,7 @@ requires = ["poetry-core", "poetry-dynamic-versioning"]
 build-backend = "poetry_dynamic_versioning.backend"
 
 [tool.poetry-dynamic-versioning]
-enable = true
+enable = false
```
Comment (asingamaneni) [inline pyproject.toml:82]: This has to b enabled, please turn it on!

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "brickflows"
-version = "0.0.1-pre.36+4a13620"
+version = "0.0.1-pre.21+152a31c"
```
Comment (asingamaneni) [inline pyproject.toml:3]: please revert this back, need not be changed

**Review by asingamaneni — APPROVED** [review]
LGTM, thanks for your contribution!

**Pull request merged**